DeepCat.js

```diff
@@ -202,17 +198,26 @@
 	}
 
 	function getSearchTerms( input ) {
-		return input.match( searchTermRegExp( keyString ) );
+		return input.match( '(-?\\b' + keyString + '\\s*'
+							+ '(?:("([^\\"]|\\.)+")' 			// search for keyword:"term including \"escaped\" quotes"
+							+ '|(?!' + keyString + ')\\S+))' 	// search for keyword:term excluding keyword term as parameter
```

Comment:
I want the elements of a deepcat search term that "belong together" stick together in the output list. Meaning: "deepcat: Minus Berlin" should result in a list 

     ["deepcat: Minus", "Berlin"]

removing the line leads to 

      ["deepcat:", "Minus", "Berlin"]